app/controllers/step_by_step_pages_controller.rb

```diff
@@ -131,11 +131,12 @@ def internal_change_notes
 
   helper_method :placeholder_values
   def placeholder_values
-    publish_time ||= Time.current.tomorrow
+    publish_time ||= Time.current.tomorrow.change(hour: 9)
     {
       day: params.dig(:schedule, :date, :day) || publish_time.day,
       month: params.dig(:schedule, :date, :month) || publish_time.month,
       year: params.dig(:schedule, :date, :year) || publish_time.year,
+      time: params.dig(:schedule, :time) || publish_time.strftime("%-l:%M%P"),
```

Comment:
Note to self - make a time helper for this, based on the one in content publisher ^